build.gradle

```diff
@@ -858,17 +857,45 @@ task product(type: Zip) {
         exclude '.git*'
       }
     }
-    if (isEnterpriseProduct && hasAqpProject) {
-      // copy enterprise shared libraries for optimized JNI calls
-      copy {
-        from aqpProject.projectDir.path + '/lib'
-        into "${snappyProductDir}/jars"
+    if (isEnterpriseProduct) {
+      if (hasAqpProject) {
+        // copy enterprise shared libraries for optimized JNI calls
+        copy {
+          from aqpProject.projectDir.path + '/lib'
```

Comment:
If the intention is to just copy only the native libs then would it be better to have some regex for those files, instead of full dir. You might not want to add other jars which you keep here ... I don't know...may be it is ok...but please think...

Reply:
There is only a single jar in respective directories. If more get added in future, we can update the script with appropriate regex.